### ISSUE TITLE
Alpha channel in .BMP files always 1 with Filetexture (DX11 2D)

### ISSUE THREAD

**Issue by mediadog** [mrvux/dx11-vvvv#303]
With both 0.7.1 and 1.0.1, all .BMP files that have alpha transparency load with no transparency.  Loading a .PNG file exported with a graphics program from that same .BMP file does show transparency.  Pipet confirms the alpha value is seen as 1 over the whole image.

I would love to attach an example patch and .bmp and .png files here, but Github absolutely refuses to take a .zip or .ZIP file no matter what I do, whether it was generated by Windoze or WinRAR.   AND I can't attach a .bmp file here!  Grrrr...

![capture-1](https://cloud.githubusercontent.com/assets/332032/24228105/66b9b562-0f48-11e7-98f4-b9abca953379.png)

**Comment by mrvux** [comment]
This goes a bit same as the 16 bits png, moving to WIC would eventually solve it, but WIC has other issues (doesn't generate mipmaps for example).

Main issue why I'm a bit wary over those is that a breaking change on filetexture is a bit like breaking every single patch, so i try to be a bit careful on that one (actually some bmp with alpha could even be in a patch as solid and become suddenly transparent, so even a fix is a break somehow).

**Comment by mediadog** [comment]
Understand about possibly "breaking" existing patches with alpha bmps, but the alternative is perpetuating a broken feature, much worse in my book.  And in this case a suddenly transparent bmp would be what I call a "good" failure, meaning it is one that would be immediately obvious and easily fixed.  (Also it would not happen unless they had explicitly turned on blending, which I only do when I am expecting to use alpha.)

Doesn't vvvv have the feature to handle such things as "legacy", so the fixed node will not break existing patches?

**Comment by mrvux** [comment]
I agree on that, but that's only one of those examples, by swapping loader need to ensure most cases are actually working properly (bmp is one amognst other). I know SlimDX  loader does the same for some dds (normalizes them), which would need a fix too.

The only legacy features are to remove a node from nodelist (with legacy, so it still gets created in old patches, and then you can use diff, which is a simple rename tool that is really limited as well)
